src/main/java/htsjdk/samtools/liftover/Chain.java

```diff
@@ -306,16 +308,19 @@ public int hashCode() {
         return result;
     }
 
+    static OverlapDetector<Chain> loadChains(final File chainFile) {
+        return loadChains(new BufferedLineReader(openFileForReading(chainFile)), chainFile.toString());
```

Comment:
As mentioned below you probably want to push the IOUtil.assertFileIsReadable check into this method.